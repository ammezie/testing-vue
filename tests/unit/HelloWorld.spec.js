import HelloWorld from "@/components/HelloWorld.vue";
import { shallowMount } from "@vue/test-utils";

describe("HelloWorld", () => {
  it("renders props.msg when passed", () => {
    const msg = "new message";

    const wrapper = shallowMount(HelloWorld, {
      propsData: { msg }
    });

    // expect(wrapper.text()).toMatch(msg);
    expect(wrapper.text()).toMatchSnapshot(msg);
  });
});
