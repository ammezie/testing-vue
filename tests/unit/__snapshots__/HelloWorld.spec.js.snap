// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld renders props.msg when passed: new message 1`] = `"new message This is a kind of paragraph"`;
